README.md

```diff
@@ -1 +1,41 @@
-# Hello-Microverse
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Hello Microverse
+
+> This is the first project on the Microverse Learning path.
+
+![screenshot](./app_screenshot.png)
```

Comment:
- [ ] [OPTIONAL] Please update the project screenshot to reflect what you have accomplished while working on your project. The screenshot gives a glimpse to the readers about what your project is about.